### ISSUE TITLE
Do you have a CSE background?

### ISSUE THREAD

**Issue by mii9000** [toddmotto/ama#17]
Hi Todd,

I went through your LinkedIn and seems you are not from traditional CSE background. So my question is how did you manage to achieve so much in such young age without CSE? Was it hands-on work experience, books maybe, dev bootcamps, etc?

Looking forward to your reply.

**Comment by toddmotto** [comment]
Hey Ibrahim! That's right I never studied CS, though I wish I had, but I'm doing alright without it. I'm debating learning CS outside of work when I get a little less busy, but I'm not sure when that'll be! :)

I honestly would say blogging has been the key factor in getting myself places. It's opened me up to conferences, companies, people and I've had many successful job offers from people finding me online and worked at great companies and also worked with great companies from it (like Intel and Rolling Stone for example). I'm completely self-taught and like that - you've gotta be a great self starter and really push your skills and limits! I read a few jQuery/JavaScript books when I first started diving into JavaScript, but most of it I worked out myself how to do - same with writing HTML5/CSS3 stuff.
